recipes/serialchemy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "serialchemy" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ef00f71153cc076709ed58ba00731768dbabbf15c237cd423df3c808b24a988
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - sqlalchemy >=1.1
```

Comment:
You don't need this during build time, you don't import `sqlalchemy` in `setup.py`. 👍

Reply:
Removed